bug in docker-compose-dev.yml

after downloading the code, npm run start-docker:dev gives error:

<pre>
ERROR: The Compose file './docker-compose-dev.yml' is invalid because:
services.manga-dev.build contains unsupported option: 'target'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mein-manga@0.0.1 start-docker:dev: `sudo docker-compose -f docker-compose-dev.yml up`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mein-manga@0.0.1 start-docker:dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/akuma/.npm/_logs/2020-12-09T17_49_39_345Z-debug.log

</pre>

--- a/docker-compose-dev.yml
+++ b/docker-compose-dev.yml
@@ -1,4 +1,4 @@
-version: '3'
+version: '3.5'
 
 services: 
     manga-dev:

